test/test-ncu.js

```diff
@@ -8,7 +8,7 @@ const tmp             = require('tmp');
 chai.use(require('chai-as-promised'));
 chai.use(require('chai-string'));
 
-process.env.NODE_ENV = 'test';
+process.env.NCU_TESTS = true;
```

Comment:
Actually, I think `NODE_ENV` is what we want here. Jest itself sets `NODE_ENV = 'test'` so there's some good precedent for using that variable to identify a test environment.